mlflow/keras.py

```diff
@@ -62,7 +67,17 @@ def load_pyfunc(model_file):
     :param model_file: Path to Keras model file.
     :return: PyFunc model.
     """
-    return _KerasModelWrapper(_load_model(model_file))
+    if K._BACKEND == 'tensorflow':
+        import tensorflow as tf
+        graph = tf.Graph()
+        sess = tf.Session(graph=graph)
+        with graph.as_default():
+            with sess.as_default():  # pylint:disable=not-context-manager
+                K.set_learning_phase(0)
```

Comment:
Can we add a comment explaining why we do this?

Reply:
Will do.